stdlib/public/core/Character.swift

```diff
@@ -293,13 +293,13 @@ extension Character : CustomDebugStringConvertible {
 extension Character {
   @_versioned
   internal var _smallUTF16 : _UIntBuffer<UInt64, Unicode.UTF16.CodeUnit>? {
-    guard case .smallUTF16(let _63bits) = _representation  else { return nil }
+    guard case .smallUTF16(let _63bits) = _representation else { return nil }
     _onFastPath()
     let bits = UInt64(Builtin.zext_Int63_Int64(_63bits))
-    let nonZeroBitCount = type(of: bits).bitWidth - bits.leadingZeroBitCount
+    let minBitWidth = type(of: bits).bitWidth - bits.leadingZeroBitCount
```

Comment:
As of SE-0104, `nonzeroBitCount` is now the name for popcount, so the name of this local variable is potentially confusing; I've renamed to aid comprehension.